src/transactions/OfferExchange.cpp

```diff
@@ -1391,4 +1454,129 @@ convertWithOffers(
         return ConvertResult::ePartial;
     }
 }
+
+struct ExchangedQuantities
```

Comment:
nit: `ExchangedQuantities` is being defined in the `stellar` namespace, should probably avoid this